config/hg19.clean.yml

```diff
@@ -1,811 +1,811 @@
 ---
```

Comment:
Github is doing something very odd here. The new file is identical to the master branch (copy/pasted from master branch directly, no formatting changes)